app/uk/gov/hmrc/economiccrimelevyregistration/cleanup/OtherEntityTypeDataCleanup.scala

```diff
@@ -23,7 +23,7 @@ import javax.inject.Inject
 class OtherEntityTypeDataCleanup @Inject() () extends DataCleanup {
   def cleanup(registration: Registration): Registration = {
     val otherEntityJourneyData = registration.otherEntityJourneyData.copy(
-      businessName = None,
+      businessName = registration.otherEntityJourneyData.businessName,
```

Comment:
Do we need this line if we are not making changes? We are copying and overriding fields where necessary.